controllers/ses.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+const DB = require("../db/DB");
+
+
+const notification = async function(req, res) {
```

Comment:
Good point ... I was just moving code around and I borrowed the expression syntax instead of declarations. I like declarations more though, so I'll update it.